### ISSUE TITLE
[a11y] [Android] Navigation bar is not visible when changing device display size while the app is open

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6839]
**Important details:** 

* the navigation bar is visible after swipe-killing the app and reopening it;

* this is also reproducible if the display size of the device is changed to default while the app is opened in background;

* tested device: Pixel 2XL;

**Affected versions:**

* 2.15.0 (2.202305041546); 

**Tested Platforms:**

* Android 11;

**Prerequisites:** 

* Have a Fx account with a subscription;

* Have Mozilla VPN app installed and open in the background;

* Be signed in;

**Steps to reproduce:**

1. On your settings device, change the Display size to the largest option available (_on Google Pixel ~~> Settings~~  Display - Advanced - Display size - Larges_t);
1. Open Mozilla VPN and observe the screen;

**Expected result:**

* There are no issues. Each element from the screen is visible;

**Actual result:**

* The navigation bar is not visible, even though the screen is scrollable; 

**Notes:**

* See the attached video;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-4723)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

I am able to reproduce this issue on the latest 2.17 version as well, using Android 13. 

Also, the navigation bar disappears if the display size is set to the smallest option available while the VPN is open in background.

Attaching a video as well. 

!screen-20230828-163643.mp4|width=1080,height=2400!
